packages/client/src/utils/rest.ts

```diff
@@ -1,11 +1,22 @@
-import { HTTP_URL } from '../lib/resources';
+export const HTTP_URL = process.env.REACT_APP_HTTP_URL;
 
-export const post = (path: string, payload: object) =>
-  fetch(`${HTTP_URL}${path}`, {
-    method: 'DELETE',
-    headers: {
-      Accept: 'application/json',
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify(payload),
-  }).then((response) => response.json());
+export const http = {
```

Comment:
Do we need `http`?

Reply:
it's a namespace 🤷‍♂️ we can change it to a better name

Reply:
I am just wondering, why do we need it though, isn't the module name (which we could rename) enough as a namespace?